management/server/user.go

```diff
@@ -691,6 +726,7 @@ func (am *DefaultAccountManager) SaveUser(accountID, initiatorUserID string, upd
 	}
 
 	// only auto groups, revoked status, and name can be updated for now
+	// when addIfNotExists is set to true the newUser will use all fields from the update input
```

Comment:
I couldn't find the changes relevant to this comment

Reply:
This is more like a remember of what happening earlier:

```go
		if !addIfNotExists {
...
		}
		// will add a user based on input
		oldUser = update
```

I will try to make it more clear